### PR TITLE
feat: add chart showing most viewed items in compose view

### LINKED ISSUE #68: in compose/teacher view, add a chart to the dashboard displaying most accessed items
in graasp compose/teacher view, viewing an item generates an action with verb `accessed`, which also includes a `target` object describing the item which was accessed, including its `displayName`. the idea is to add a chart to the compose/teacher view dashboard which displays the most accessed items within a space, i.e. `displayName` on the x-axis and # of views on the y.

### PR DESCRIPTION
**Pull request by hagoptaminian**
closes #68

### CODE REVIEW

**Comment by hagoptaminian** [comment]
Chart looks like this:

Screen Shot 2021-03-09 at 3 37 49 PM

File: src/components/space/charts/ItemsByAccessedCount.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+      <ResponsiveContainer width="95%" height={CONTAINER_HEIGHT}>
+        <BarChart
+          data={formattedItemsCount}
+          margin={{ top: 30, bottom: 20, left: 20, right: 20 }}
+        >
+          <CartesianGrid strokeDasharray="2" />
+          <XAxis dataKey="displayName" tick={{ fontSize: 12 }} />
+          <YAxis tick={{ fontSize: 14 }} domain={[0, yAxisMax]} />
+          <Tooltip />
+          <Bar dataKey="count" name="Count" fill={theme.palette.primary.main} />
```
Comment (juancarlosfarah) [inline src/components/space/charts/ItemsByAccessedCount.js:73]: Do we need to translate `Count`?

**Review by juancarlosfarah — APPROVED** [review]
@hagoptaminian, very nice PR. Clean. Well structured. Going to approve and merge. Just a question there about translation, but you can patch it later. 🔥

File: src/components/space/charts/ItemsByAccessedCount.js (hunk 1 above)
Comment (hagoptaminian) [reply src/components/space/charts/ItemsByAccessedCount.js:73]: Yes, definitely. Good catch. Here and in the two other bar charts.

**Pull request merged**